typescript/sequelize/README.md

```diff
@@ -87,49 +87,163 @@ npm run start
 
 The example contains comments explaining the code and the operations being performed.
 
-## Sequelize Pet Clinic with DSQL
+## Usage notes
 
-### Connect to an Aurora DSQL cluster
+### Connecting to DSQL
 
-The example uses the `AuroraDSQLClient` from the [Aurora DSQL Connector for node-postgres](https://github.com/awslabs/aurora-dsql-nodejs-connector/blob/main/packages/node-postgres/README.md), which automatically handles IAM token generation for each connection. The `dialectModule` option allows us to inject a custom `pg` module with the DSQL-aware client. When using Sequelize with the Postgres dialect option, Sequelize uses [node-postgres](https://node-postgres.com/) to connect.
+DSQL is PostgreSQL-compatible, so use the `postgres` dialect. The [Aurora DSQL Connector for node-postgres](https://github.com/awslabs/aurora-dsql-nodejs-connector/blob/main/packages/node-postgres/README.md) handles IAM token generation automatically. Inject it via the `dialectModule` option.
```

Comment:
> Inject it via the `dialectModule` option.

Nit: Can we be a little more verbose here? When I first read it I thought "it" was referring to the dialect, not the connector. I think just specify what's injected and where.